websites/A/AnimeHeaven/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "AnimeHeaven",
+  "author": {
+    "name": "Tenri",
+    "id": "1999"
```

Comment:
Didn't know snowflake like this exist

Reply:
Sorry, thought it was a discriminator I needed to put in. I'll fix that